### PR TITLE
Orderly Tweaks & Nurse Orderlies

### PR DESCRIPTION
**Pull request by Apogee-dev**
Changes Nurse to an alt title for Orderly instead of Medical Doctor.

Additionally, adjusts the Orderly's starting equipment. Starting paralytic injector count has been reduced to 5 and the medipop dispenser has been changed to a cheaper version that dispenses lollipops with 1u Piccolyn instead of 10u Tricordrazine.

:cl:
 * tweak: Reduced starting injector count for Orderlies
 * tweak: Orderlies now spawn with cheaper lollipop dispensers
 * tweak: Changed Nurse to an alt title for Orderly

### CODE REVIEW

**Comment by Optimism333** [comment]
This concept pleases me greatly. Finally nurses can be in charge of cleaning medbay, updating records, and overall keeping patients in good spirits.

I have a few suggestions to make the nurse alt-title stand out a bit more compared to an Orderly:
- No paralytic injectors (Unlike the Orderly I don't think they should have the duty of being a bouncer)
- They should spawn instead with a few items aimed at improving a patient's experience while in medbay. A few ideas could be as follows: a bottle of tramadol pills (in case patients arrive in severe pain), a pack of NT Standard cigarettes (lighter included?), and an ice cream cone dispenser instead of a lollipop dispenser (maybe it needs to be refilled with nutriment after a few uses).

If one or any of the suggestions above could be considered I'd appreciate it.

**Comment by Apogee-dev** [comment]
> 
> 
> This concept pleases me greatly. Finally nurses can be in charge of cleaning medbay, updating records, and overall keeping patients in good spirits.
> 
> I have a few suggestions to make the nurse alt-title stand out a bit more compared to an Orderly:
> 
>     * No paralytic injectors (Unlike the Orderly I don't think they should have the duty of being a bouncer)
> 
>     * They should spawn instead with a few items aimed at improving a patient's experience while in medbay. A few ideas could be as follows: a bottle of tramadol pills (in case patients arrive in severe pain), a pack of NT Standard cigarettes (lighter included?), and an ice cream cone dispenser instead of a lollipop dispenser (maybe it needs to be refilled with nutriment after a few uses).
> 
> 
> If one or any of the suggestions above could be considered I'd appreciate it.

I considered it, but with only a couple exceptions alt titles don't generally spawn with wildly different equipment from their base job, just the outfit change. Nurse and Orderly still have the same duties and both jobs do come with the equipment to handle both patient welfare and... well, order, so it doesn't seem too necessary to me

**Comment by Optimism333** [comment]
> > This concept pleases me greatly. Finally nurses can be in charge of cleaning medbay, updating records, and overall keeping patients in good spirits.
> > I have a few suggestions to make the nurse alt-title stand out a bit more compared to an Orderly:
> > ```
> > * No paralytic injectors (Unlike the Orderly I don't think they should have the duty of being a bouncer)
> > 
> > * They should spawn instead with a few items aimed at improving a patient's experience while in medbay. A few ideas could be as follows: a bottle of tramadol pills (in case patients arrive in severe pain), a pack of NT Standard cigarettes (lighter included?), and an ice cream cone dispenser instead of a lollipop dispenser (maybe it needs to be refilled with nutriment after a few uses).
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > If one or any of the suggestions above could be considered I'd appreciate it.
> 
> I considered it, but with only a couple exceptions alt titles don't generally spawn with wildly different equipment from their base job, just the outfit change. Nurse and Orderly still have the same duties and both jobs do come with the equipment to handle both patient welfare and... well, order, so it doesn't seem too necessary to me

Well, I won't press the issue then. Excitement slightly deflated, but what can you do. Your point on the alt-titles not getting greatly different equipment is fair.

My only other suggestion is that I think four injectors is plenty to get on round-start for those last resort shitter removal situations. Five is still a lot in my opinion.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -5090,6 +5090,71 @@
 	if(!H.isDead())
 		updateconsumingalt(equipped_slot == slot_wear_mask)
 
+/obj/item/weapon/reagent_containers/food/snacks/lollicheap
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: make a subtype instead of pasting all this!

**Comment by Kurfursten** [comment]
Less injectors is a big who cares, most of the time I use two at most (but it's nice to have the security of more stashed somewhere, I never keep my extras with me)
New alt title also couldn't care either way
But not my tricord lollipops! The borgs get them, why not orderlies? Tricord is trivially easy to make and doesn't really heal much anyway.

**Comment by Apogee-dev** [comment]
> 
> 
> Less injectors is a big who cares, most of the time I use two at most (but it's nice to have the security of more stashed somewhere, I never keep my extras with me) New alt title also couldn't care either way But not my tricord lollipops! The borgs get them, why not orderlies? Tricord is trivially easy to make and doesn't really heal much anyway.

The borgs get the tricord lollipops as a nice little bonus to make up for the fact that they're still generally worse at medical than a real doctor, imo. Tricord being so easy to make, it's not like the orderly lost a valuable source of the stuff... and putting piccolyn in them is funnier, if you ask me.
I might revert it back to a normal medipop dispenser and maybe add 1u piccolyn to the regular medipops if the other orderly players want, though. It's not hugely impactful either way.

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (Apogee-dev) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: I'll probably have to refactor it for the normal medipops too, honestly. Which wouldn't be a terrible idea, in retrospect.

**Comment by gurfan** [comment]
i always wanted to be knocked out and put in a straitjacket by a nurse

**Comment by jwhitak** [comment]
gurfan, the only one brave enough to say what everyone else is thinking...

I don't feel that orderlies having the tricord pops is too strong, but we should compare to other departments having boxes of limited DONK.  Also, thanks doc.

**Comment by Apogee-dev** [comment]
> 
> 
> gurfan, the only one brave enough to say what everyone else is thinking...
> 
> I don't feel that orderlies having the tricord pops is too strong, but we should compare to other departments having boxes of limited DONK. Also, thanks doc.

Yeah, considering medbay has access to unlimited tricord courtesy of chemistry existing it's not like having a source of it is THAT big of a deal, unless somebody steals the dispenser. Changing it to piccolyn is more of a flavor decision, and i think the flavor of YOU DESERVE A MEDAL DOC is a lot funnier

**Comment by Eneocho** [comment]
Just add 1u tricord to the lollipops too so you can use them both to top the guy off and to get the YOU THE MAN, DOC experience.

**Comment by Apogee-dev** [comment]
> 
> 
> Just add 1u tricord to the lollipops too so you can use them both to top the guy off and to get the YOU THE MAN, DOC experience.

**Comment by Apogee-dev** [comment]
> 
> 
> Just add 1u tricord to the lollipops too so you can use them both to top the guy off and to get the YOU THE MAN, DOC experience.

FUCKING misclicked, anyway, sure why not, 1u is barely anything anyway

**Comment by SonixApache** [comment]
femsec buff

**Comment by Optimism333** [comment]
Upon further thought, I still feel pretty strongly that five autoinjectors is a bit much, particularly if they're meant to be a last resort for dealing with trouble in medbay when reasoning hasn't worked and security is unavailable.

I've been on the receiving end of an overly aggressive Orderly hunting me down outside medbay and even after stunning me multiple times he still had injectors leftover. I would really prefer that the motto with those items be "less is more". They're very useful, and I doubt that even if it was three per Orderly that there'd be a danger of running out if the injectors were used conservatively in emergencies.

I settled on the number four as a target because I asked the guy who hunted me as Orderly that round (we made up afterwards) how many he thought was reasonable, and he said four.

**Comment by Apogee-dev** [comment]
> 
> 
> Upon further thought, I still feel pretty strongly that five autoinjectors is a bit much, particularly if they're meant to be a last resort for dealing with trouble in medbay when reasoning hasn't worked and security is unavailable.
> 
> I've been on the receiving end of an overly aggressive Orderly hunting me down outside medbay and even after stunning me multiple times he still had injectors leftover. I would really prefer that the motto with those items be "less is more". They're very useful, and I doubt that even if it was three per Orderly that there'd be a danger of running out if the injectors were used conservatively in emergencies.
> 
> I settled on the number four as a target because I asked the guy who hunted me as Orderly (we made up afterwards) how many he thought was reasonable, and he said four.

yeah, i'm not 100% on injector count. I think three might be too few, particularly if things get hectic, but four seems reasonable. You aren't the only player I've heard that number from, so I'll tune it down just a bit more.

**Comment by Optimism333** [comment]
> > Upon further thought, I still feel pretty strongly that five autoinjectors is a bit much, particularly if they're meant to be a last resort for dealing with trouble in medbay when reasoning hasn't worked and security is unavailable.
> > I've been on the receiving end of an overly aggressive Orderly hunting me down outside medbay and even after stunning me multiple times he still had injectors leftover. I would really prefer that the motto with those items be "less is more". They're very useful, and I doubt that even if it was three per Orderly that there'd be a danger of running out if the injectors were used conservatively in emergencies.
> > I settled on the number four as a target because I asked the guy who hunted me as Orderly (we made up afterwards) how many he thought was reasonable, and he said four.
> 
> yeah, i'm not 100% on injector count. I think three might be too few, particularly if things get hectic, but four seems reasonable. You aren't the only player I've heard that number from, so I'll tune it down just a bit more.

Thanks.

**Comment by BomberBro** [comment]
More jobs absorbed into the sec/assistant gestalt.

**Comment by Eneocho** [comment]
>More jobs absorbed into the sec/assistant gestalt.

`>Station Time: 8.766e+9:00`
`>Visions of space in the distant future: a red administers a beating to a grey for meandering within 6.2 meters of the head of personnel's office.`
`>The red is not aware of the grey's presence, its sensory organs are vestigal; its actions are driven by instinct and muscle memory alone. There are no variables within the self-sustaining amnion that insulates Space Station 13 from the outside universe; all it knows intrinsically is all that will come to pass- thinking is simply obsolete.`
`>The grey nociceives the savaging, yet with what strength it can muster it leans into the beating. This is not out of longing for any release, after its carcass serves a sentence of 1.5e+8 minutes it will eventually be dragged to the cloning vats, where it will be processed with 2.3496e-21 percent greater efficiency than the last time.`
`>This routine, which for these genetically degenerate unhumans once was life, has played out thousands of times before- and will again thousands more, and neither can care.`

**Pull request merged**